types/schemas/digitalPlanningApplication/enums/PropertyTypes.ts

```diff
@@ -27,14 +27,14 @@ export const PropertyTypes = {
     'Public / Village Hall / Other Community Facility',
   'commercial.community.hall.club': 'Youth Recreational / Social Club',
   'commercial.community.wc': 'Public Convenience',
-  'commercial.community.cemetary':
+  'commercial.community.cemetery':
     'Cemetery / Crematorium / Graveyard. In Current Use.',
-  'commercial.community.cemetary.columbarium': 'Columbarium',
-  'commercial.community.cemetary.crematorium': 'Crematorium',
-  'commercial.community.cemetary.chapelOfRest': 'Chapel Of Rest',
-  'commercial.community.cemetary.cemetary': 'Cemetery',
-  'commercial.community.cemetary.military': 'Military Cemetery',
-  'commercial.community.cemetary.mortuary': 'Mortuary',
+  'commercial.community.cemetery.columbarium': 'Columbarium',
+  'commercial.community.cemetery.crematorium': 'Crematorium',
+  'commercial.community.cemetery.chapelOfRest': 'Chapel Of Rest',
+  'commercial.community.cemetery.cemetery': 'Cemetery',
```

Comment:
This [does match the spreadsheet](https://docs.google.com/spreadsheets/d/1ePihRD37-2071Wq6t2Y7QtBt7juySWuVP6SAF6T-0vo/edit?gid=769589103#gid=769589103), but is this actually correct?

Reply:
Cleaning up our own typos across project types, res units, and constraints seems right to me - but I'm honestly less sure about property types since these are essentially externally sourced / setup to match classification codes returned from OS ?